gc/gc-tool/src/main/java/org/projectnessie/gc/tool/cli/commands/JdbcCreateSchema.java

```diff
@@ -39,9 +39,12 @@ public class JdbcCreateSchema extends BaseCommand {
   @Override
   protected Integer call(Closeables closeables) throws Exception {
     DataSource dataSource = closeables.maybeAdd(jdbc.createDataSource());
+    SchemaCreateStrategy schemaCreateStrategy = jdbc.getSchemaCreateStrategy();
+    if (schemaCreateStrategy == null) {
```

Comment:
You can omit this null-check by specifying a default-value for the option. ANd tell picocli to always show the default value

Reply:
I wanted the default value to be null in order to not introduce a behavioral change. Currently:

1. `create-schema` assumes `CREATE` if this parameter is null
2. `gc` assumes nothing if this parameter is null (no schema creation).

This is equivalent to the current behavior. Are we OK to change this?

Reply:
Ah, okay - so that option is used from two commands.